src/main/kotlin/uk/gov/justice/hmpps/offendersearch/dto/OffenderUserAccess.kt

```diff
@@ -19,8 +19,6 @@ package uk.gov.justice.hmpps.offendersearch.dto
  */
 data class OffenderUserAccess(
     val username: String? = null,
-    val allowWhenExclusionMatched: Boolean = false,
```

Comment:
ended up these scopes behaved the same way as the two below, so I got rid and deleted associated tests etc